pyproject.toml

```diff
@@ -31,7 +31,7 @@ ipdb = "^0.13.9"
 sphinx = ">=5.0.1"
 qdrant-sphinx-theme = { git = "https://github.com/qdrant/qdrant_sphinx_theme.git", branch = "master" }
 black = "^22.3.0"
-torchvision = "^0.12.0"
+torchvision = ">=0.12.0"
```

Comment:
why

Reply:
It was fixing torchvision to 0.12.x (latest = 0.15.x) and the latest poetry couldn't verify its hash on CI, essentially braking our tests.

Reply:
yeah, my bad, I forgot that `^x.y.z` does not allow to increase `y` if `x == 0`

Reply:
Thanks!